steward/src/commands/cosmos_mode.rs

```diff
@@ -26,6 +34,16 @@ impl Runnable for CosmosSignerCmd {
         let config = APP.config();
         info!("Starting application");
         abscissa_tokio::run(&APP, async {
+            if let Err(err) = CORK_QUERY_CLIENT.set(CorkClientWrapper::new().await.unwrap()) {
+                // SetError<T> does not implement Debug and therefore can't call unwrap() or expect()
+                panic!("{}", err);
+            }
```

Comment:
Wow, that's a really weird deficiency in `SetError`

Reply:
right?